From 5009415a801185ac2991eb44723446eef82fee73 Mon Sep 17 00:00:00 2001
From: Dan Nica <dan.nica@nxp.com>
Date: Wed, 2 Dec 2020 11:35:22 +0200
Subject: [PATCH 12/76] s32-gen1: ddrss: Define an empty store_csr function

The latest versions of the DDR Tool generated code no longer
define a weak store_csr function, therefore we have to define one
explicitly.

Issue: ALB-5963
Signed-off-by: Dan Nica <dan.nica@nxp.com>
---
 arch/arm/cpu/armv8/s32/s32-gen1/soc.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/cpu/armv8/s32/s32-gen1/soc.c b/arch/arm/cpu/armv8/s32/s32-gen1/soc.c
index 05d7e13d5b..5b2dd88488 100644
--- a/arch/arm/cpu/armv8/s32/s32-gen1/soc.c
+++ b/arch/arm/cpu/armv8/s32/s32-gen1/soc.c
@@ -159,6 +159,10 @@ extern struct ddrss_conf ddrss_conf;
 extern struct ddrss_firmware ddrss_firmware;
 #endif
 
+#if defined(CONFIG_SYS_FSL_DDRSS) && defined(CONFIG_S32_STANDALONE_BOOT_FLOW)
+void store_csr(void) {}
+#endif
+
 #ifdef CONFIG_S32_SKIP_RELOC
 __weak int dram_init(void)
 {
-- 
2.17.1

